crates/fpe/src/wasm_bindgen/integer.rs

```diff
@@ -0,0 +1,76 @@
+use num_bigint::BigUint;
+use num_traits::Num;
+use wasm_bindgen::{prelude::wasm_bindgen, JsValue};
+
+use crate::core::{Integer, KEY_LENGTH};
+
+fn fpe(
+    input: &str,
+    radix: u32,
+    digits: usize,
+    key: Vec<u8>,
+    tweak: Vec<u8>,
+    encrypt_flag: bool,
+) -> Result<String, JsValue> {
+    // Ensure that the key length is correct
+    if key.len() != KEY_LENGTH {
+        return Err(JsValue::from_str(&format!(
+            "FPE Big Integer error: key length incorrect: {}, expected {}",
+            key.len(),
+            KEY_LENGTH
+        )));
+    }
+
+    // Copy the key bytes into a 32-byte array
+    let mut k: [u8; 32] = [0; 32];
+    k.copy_from_slice(&key);
```

Comment:
Use `TryFrom` implementation

Reply:
You can also remove the condition above with the `try_into`

Reply:
done